### PR TITLE
Menu buttons fix

### PR DESCRIPTION
**Pull request by deplanty**
- Set menu buttons settings in the config file;
- Use the config file to set the buttons;
- Set all the background colors with one parameter : `bg`.

### CODE REVIEW

File: src/button/button.py [hunk 1]
```diff
@@ -71,35 +74,49 @@ def __setKwargs(self, **kwargs):
...
+            state = kwargs.pop("state")
+            if state == "disabled":
+                self.state = State.disabled
+            elif state == "enabled":
+                self.state = State.normal
+        if "bg" in kwargs:
+            bg = kwargs.pop("bg")
+            self.bg_normal = bg
+            self.bg_hover = bg
+            self.bg_pressed = bg
```
Comment (Hboni) [inline src/button/button.py:112]: I think you forgot to use the index of `bg` list?

File: src/button/button.py [hunk 2]
```diff
@@ -71,35 +74,49 @@ def __setKwargs(self, **kwargs):
...
 
         self.__dict__.update(**kwargs)
         # Update text_core
         if "text_font" in kwargs:
             self.text_core = pygame.font.Font(**self.text_font)
+
```
Comment (Hboni) [inline src/button/button.py]: Too much whitespace.

File: src/button/button.py [hunk 3]
```diff
@@ -154,11 +171,13 @@ def set(self, **kwargs):
         :param 3-tuple bg_normal: color for normal background
         :param 3-tuple bg_hover: color for background when hover
         :param 3-tuple bg_pressed: color for background when pressed
+        :param 3-tuple bg: color for all the backgrouds
```
Comment (Hboni) [inline src/button/button.py]: Forgot `n` un `backgrounds`

**Review by Hboni — CHANGES_REQUESTED** [review]
Little changes, but yeah it's not a good idea to use feature branch to create another one, we can see commit from another branch here ... BAD BOY !

File: src/button/button.py (hunk 1 above)
Comment (deplanty) [reply src/button/button.py:112]: I don't understand the question... Which index?

Comment (Hboni) [reply src/button/button.py:112]: `bg[0]`, `bg[1]`, ...
`bg` is the entire list, don't you need to choose the right one for each background?

Comment (deplanty) [reply src/button/button.py:112]: It's meant to apply the same background color to every backgrounds!
`Button(bg=(10, 10, 50))` > `bg_normal = bg_hover = bg_pressed = bg`

Comment (Hboni) [reply src/button/button.py:112]: So, for now, the different backgrounds are always the same?

Comment (Hboni) [reply src/button/button.py]: So, for now, the different backgrounds are always the same?

Comment (deplanty) [reply src/button/button.py:112]: No, but you have a simple solution to do it :smile:
If you want a pressed background of color C1 and all the other backgrounds of color C2:
`Button(bg=C2, bg_pressed=C1)`
If you want all the backgrounds of color C1:
`Button(bg=C1) == Button(bg_normal=C1, bg_hover=C1, bg_pressed=C1)`

Comment (Hboni) [reply src/button/button.py:112]: Ah ok, I didn't think about that type of use!

**Pull request closed without merging**